test/metabase/driver/mysql_test.clj

```diff
@@ -468,15 +477,21 @@
             (sync/sync-table! table)
             (let [field (db/select-one Field :table_id (u/id table) :name "json_bit → 1234")
                   compile-res (qp/compile
-                                {:database (u/the-id (mt/db))
-                                 :type     :query
-                                 :query    {:source-table (u/the-id table)
-                                            :aggregation  [[:count]]
-                                            :breakout     [[:field (u/the-id field) nil]]}})]
-              (is (= (str "SELECT convert(json_extract(json.json_bit, ?), UNSIGNED) AS `json_bit → 1234`, "
-                          "count(*) AS `count` FROM `json` GROUP BY convert(json_extract(json.json_bit, ?), UNSIGNED) "
-                          "ORDER BY convert(json_extract(json.json_bit, ?), UNSIGNED) ASC")
-                     (:query compile-res)))
+                               {:database (u/the-id (mt/db))
+                                :type     :query
+                                :query    {:source-table (u/the-id table)
+                                           :aggregation  [[:count]]
+                                           :breakout     [[:field (u/the-id field) nil]]}})]
+              (is (= ["SELECT"
```

Comment:
I updated a few of these tests to format the SQL and split the lines so it's easier to diff -- it was very hard to squint at one giant blob of SQL and figure out what had changed